theano/tensor/basic.py

```diff
@@ -6093,3 +6093,12 @@ def c_code_cache_version(self):
 
     def do_constant_folding(self, node):
         return False
+
+    def connection_pattern(self, node):
+        return [[False] for i in node.inputs]
+
+    def grad(self, inputs, grads):
+        return [DisconnectedType()() for i in inputs]
+
+    def R_op(self, inputs, eval_points):
+        return [zeros(inputs, self.dtype)]
```

Comment:
For the moment, I don't think it is a priority, and it would still be a bit of work to make sure the fix is correct, and to add the required tests.